vertx-mutiny-code-generator/src/main/java/io/smallrye/mutiny/vertx/apigenerator/analysis/VertxGenAnalysis.java

```diff
@@ -65,10 +65,15 @@ public AnalysisResult analyze() {
         modules.add(new WriteStreamModule());
 
         List<ShimClass> shims = new ArrayList<>();
+
+        for (VertxGenInterface anInterface : generator.getCollectionResult().interfaces()) {
+            logger.debug("Interface {} has been collected", anInterface.getFullyQualifiedName());
+        }
         for (VertxGenInterface itf : generator.getCollectionResult().interfaces()) {
             logger.debug("Analyzing {}", itf.getFullyQualifiedName());
             ShimClass shim = new ShimClass(itf);
             for (ShimModule module : modules) {
+                logger.debug("Analyzing module {}", module.toString());
```

Comment:
I don't think you need `.toString()` here since it will be implicitly be made by the logger.